README.md

```diff
@@ -1,12 +1,66 @@
-# Accessible diff viewer
+---
 
-The repository contains a work in progress diff viewer with an emphasis on being accessible.
+# Diffcessible: An accessible diff viewer
 
-If you want to try it out:
+Diffcessible is an terminal based diff viewer with an emphasis on being accessible. Our goal is to build a diff viewer on top of Git diff that is accessible to screen readers.
+
+## Overview
+
+Diffcessible currently has two main components:
+
+1. `bin/main.ml`: The main entry point for the application. It is responsible for parsing the command-line arguments and calling the appropriate functions to display the diff. The key point to note here is that the command-line requires a path to a diff file as a required positional argument. See the section on [Running the Project](#running-the-Project) for more details.
+
+2. `lib/interactive_viewer.ml`: The main logic for the diff viewer. It is responsible for parsing the diff and displaying it in the terminal. This fetches the hunk information from the diff and displays it in a user-friendly manner by indexing the hunks. The core logic for the viewer is implemented in the `view` function, which allows the user to navigate through the hunks using the `n` and `p` keys to move to the next and previous hunks, respectively. The `q` key can be used to quit the viewer.
+
+## Installation
+
+To set up the project, please fork the repository and clone it to your local machine. You will need to have `opam` installed to install the dependencies. Once you have `opam` installed, you can run the following command to install the dependencies:
+
+```bash
+opam install . --deps-only --with-test
 ```
-# Install dependencies. Requires having opam installed
-$ opam install --deps-only .
 
-# launch the binary
-$ dune exec bin/main.exe
+## Running the Project
+
+To run the project, you can use the following commands:
+
+```bash
+dune exec diffcessible [path/to/file1]
 ```
+
+## Roadmap
+
+As this project is in its early stages, we have a few features that we would like to implement:
+
+- [ ] Side-by-side diff viewer
+- [ ] Color coding for added, removed, and modified lines
+- [ ] Display metadata about the diff and more information on the status bar
+- [ ] Binding `j` and `k` for scrolling
+
+## Technologies Used
+
+The following is a list of the technologies used in the project:
+
+- **Patch Library:** For parsing and printing diffs.
```

Comment:
It would be nice to have links to each mentioned library repository in this section.

Reply:
Resolved.